docs/license.rst

```diff
@@ -0,0 +1,132 @@
+.. _license-page:
+
```

Comment:
you will also need to add this to index.html to appear as a tab on readthedocs.

Reply:
Not using `license.rst` anymore